ts/components/CalendarsListContainer.tsx

```diff
@@ -0,0 +1,146 @@
+import { Button, List, Text } from "native-base";
+import * as React from "react";
+import RNCalendarEvents, { Calendar } from "react-native-calendar-events";
+import { connect } from "react-redux";
+
+import * as pot from "italia-ts-commons/lib/pot";
+import { EdgeBorderComponent } from "../components/screens/EdgeBorderComponent";
+import ListItemComponent from "../components/screens/ListItemComponent";
+
+import I18n from "../i18n";
+import { preferredCalendarSaveSuccess } from "../store/actions/persistedPreferences";
+import { Dispatch } from "../store/actions/types";
+import { GlobalState } from "../store/reducers/types";
+
+type OwnProps = {
+  onCalendarSelected: (calendar: Calendar) => void;
+  onCalendarLoaded: () => void;
```

Comment:
Since we could have an error while calendars are loading, is it a good idea to add an optional handler:
`onCalendarError?: (error: FetchError) => void`